content/concepts/kata/translations.md

```diff
@@ -5,73 +5,124 @@ sidebar: explanation-kata
 
-## Pending translations
+## Reviewing and working with pending translations
 
-TBD
+When a translation is published, it's put into a pending state and awaits for review and approval. Sufficiently privileged users can search for them, review them, and request some fixes, approve, or reject them. Currently, pending translations can be found in one of following ways:
 
-- `Translations` tab on a kata
-- `Translations` page available to "mods"
+- Reading through the kata discourse and searching for posts where translators announced the fact that they published a translation. Translators are encouraged to post a message labelled as **Suggestion** in the kata discourse with a link and information about the language of the translation, so others could find it easily.
+- Navigating to `https://www.codewars.com/kata/kata-id/translations` page lists all accepted and pending translations of a kata.
+- A small group of users has been granted the privilege of accessing the page with a list of all pending translations eligible for approval by them. This is the most convenient way of searching for pending translations, but not available to every user at this moment.
 
-## Reviewing a translation
+### Reviewing a translation
 
-TBD
+Before a pending translation is accepted and a new language becomes available for a kata, it needs to be reviewed and determined whether it holds up to Codewars quality standards. When reviewing a translation, users can perform the following steps:
 
-- HOWTO properly review
+- **Reading the code and description**.
+- **Running the tests** against _other_ solutions along with the reference solution. Unfortunately, testing solutions to a proposed translation other than the reference solution is currently rather cumbersome. One possibility is to fork the translation, replace the reference solution with your solution, run the tests and discard the fork afterwards. Another way is to create a new kumite from scratch and copy/paste code snippets from the translation under review to your kumite.
+- **Reading the comments** to check if there are any remarks from other reviewers and whether they still hold or were addressed appropriately. Unfortunately, comments under translations cannot be labelled as an **Issue** or **Suggestion** so there are no additional markers for the severity of the comment.
+- **Verifying** if the translation holds to [translation authoring guidelines (TODO)]().
 
-## Aproving a translation
+After a review is completed, the reviewer can finalize it with one of following actions:
 
-TBD
+- **Leaving a comment** with remarks, suggestions and issues which should be addressed by the translator.
+- **Fixing problems found during the review**. The translation fork may be edited directly only by its author. If a user wants to make changes to a translation fork created by another user, they have to fork it first, edit the code, and publish the changes as a new fork.
```

Comment:
~~warning, a reviewer cannot "edit" the translation, he can just create a new fork of it.~~

ok, I let my first comment to show that the order of the sentence needs a change... x)

=> "title. Only the author of the translation ..."